chia/policy/fee_estimator.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+from typing_extensions import Protocol
+
+from chia.consensus.block_record import BlockRecord
+from chia.policy.fee_estimation import FeeMempoolInfo
+from chia.types.mempool_item import MempoolItem
+from chia.util.ints import uint64
+
+
+class FeeEstimatorConfig:
```

Comment:
empty class ?
its not referenced anywhere

Reply:
Ah, I moved this to being a dict, because different implementations will have different configs. Removed, Thanks!